server/src/main/java/org/elasticsearch/common/settings/AbstractScopedSettings.java

```diff
@@ -564,7 +564,11 @@ void validate(
                 }
             }
         }
-        setting.get(settings);
+        if (validateDependencies) {
+            setting.get(settings);
+        } else {
+            setting.validateWithoutDependencies(settings);
+        }
```

Comment:
This change and the associated change in `SecureSettings` and tests have been split into #70144